### PR TITLE
BETA: Register devendra.is-a.dev

### PR DESCRIPTION
**Pull request by devendrakanojiya**
Added `devendra.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/devendra.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devendrakanojiya",
+        "email": "s_36100@bkbirlacollegekalyan.com"
+    },
+    "record": {
+        "CNAME": "devendrakanojiya.github.io/port"
```
Comment (Stef-00012) [inline domains/devendra.json:7]: Invalid CNAME record (cannot contain `/`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/devendra.json (hunk 1 above)
Comment (devendrakanojiya) [reply domains/devendra.json]: Check now, resolved it 🙂

**Comment by Stef-00012** [comment]
seems good now

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**